pkg/rancher-desktop/main/mainmenu.ts

```diff
@@ -246,7 +246,6 @@ function setZoomLevel(focusedWindow: Electron.BrowserWindow | undefined, zoomLev
     const desiredZoomLevel = zoomLevelAdjustment === 0 ? zoomLevelAdjustment : currentZoomLevel + zoomLevelAdjustment;
 
     webContents.setZoomLevel(desiredZoomLevel);
-    focusedWindow.getBrowserView()?.webContents.setZoomLevel(desiredZoomLevel);
```

Comment:
`focusedWindow.getBrowserView()` is just `view` (from line 243), and if we get into this block we already know it's nullish from line 245. So this can never do anything.